README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+
+# About Me
+My name is **Anastasiya Apalkova** and I am a student in the MAPSS program concentrating in psychology. 
+I was born in Ukraine and moved to Chicago with my family when I was seven years old. 
+***
```

Comment:
Hi Anastasiya! It is great using *** to create a space between lines. In my assignment, I used double space and works well too!